CC/CMakeLists.txt

```diff
@@ -54,14 +50,15 @@ if (COMPILE_CC_CORE_LIB_WITH_TBB)
 	set_property( TARGET ${PROJECT_NAME} APPEND PROPERTY COMPILE_DEFINITIONS USE_TBB )
 endif()
 
-if ( COMPILE_CC_CORE_LIB_WITH_QT )
+if (COMPILE_CC_CORE_LIB_WITH_QT)
```

Comment:
Shouldn't this whole section be removed?

Reply:
no, because there are still some QT artifacts, especially the use of QTBitArray somewhere